app/uk/gov/hmrc/eoricommoncomponent/frontend/views/check_your_details_register.scala.html

```diff
@@ -382,21 +423,56 @@
   )
 }
 
+@details = @{
+    Seq(subscription.contactDetails.map { contactDetails =>
+        summaryListRow(
+                key = messages("cds.form.customs-contact-address"),
+                value = Some(contactDetailsHtml(contactDetails)),
+                call = Some(uk.gov.hmrc.eoricommoncomponent.frontend.controllers.routes.ContactDetailsController.reviewForm(service))
+            )
+        }
+    ).flatten
+}
+
 @layout_di(messages("cds.form.check-answers"), displayBackLink = false) {
     <div>
         <h1 class="govuk-heading-l">@messages("cds.form.check-answers")</h1>
+        <div>
+            <h2 class="govuk-heading-m">@headerTitle</h2>
+            @govukSummaryList(SummaryList(
+            rows =
+            individualNameDob ++
+            organisationName ++
+            organisationUtr ++
+            customsId ++
+            individualUtr ++
+            registeredAddress ++
+            dateOfEstablishment ++
+            sicCodeDisplay ++
+            summary ++
+            shortName
+            ))
+        </div>
+        @if(!isIndividual){
+        <div>
+            <h2 class="govuk-heading-m">@messages("cds.form.check-answers-vat-details")</h2>
```

Comment:
Can we check for the headers in the test?

Reply:
The missing tests to verify the headers are  now added.